eth-contracts/contracts/service/ServiceProviderFactory.sol

```diff
@@ -348,9 +349,62 @@ contract ServiceProviderFactory is RegistryContract {
         return spId;
     }
 
-  /**
-   * @notice Update service provider balance
-   */
+    function addServiceType(
+        bytes32 _serviceType,
+        uint _serviceTypeMin,
+        uint _serviceTypeMax
+    ) external
+    {
+      require(
```

Comment:
not blocker but we should move this repeated require code to a modifier. added a task to refactor doc

Reply:
noted